src/middlewared/middlewared/plugins/s3.py

```diff
@@ -54,8 +54,8 @@ async def config_extend(self, s3):
         Str('bindip'),
         Int('bindport', validators=[Range(min=1, max=65535)]),
         Int('console_bindport', validators=[Range(min=1, max=65535)]),
-        Str('access_key', max_length=20),
-        Str('secret_key', max_length=40),
+        Str('access_key', max_length=20, validators=[Match(r'^[^"]*$')]),
```

Comment:
Can we add an `explanation=` kwarg to the `Match` validator so the end-user has some idea on why this would fail if a double-quote was added? (i.e. "double-quotes are not allowed")